packages/synapse-client/package.json

```diff
@@ -7,19 +7,20 @@
   "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "default": "./dist/index.js",
-      "types": "./dist/index.d.ts"
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
```

Comment:
These were in the wrong order (per a warning when running scripts in this project)